### ISSUE TITLE
cross compile libjansi.so for aarch64

### ISSUE THREAD

**Issue by VallariTrivedi** [fusesource/jansi#100]
**I want to run Jansi on aarch64 platform**
I could compile using mvn

1. mvn -D target=linux-aarch64 install

libjansi.so is created for freeBSD-32/64; Linux-32/64 Windows-32/64

How do I cross-compile Jansi, so as to create libjansi.so for aarch64

**Comment by tomasjura** [comment]
I downloaded source of jansi-native. I tried to mvn to cross compile for Solaris, but unsuccessfully. Then I went to directory  `jansi-native/target/generated-sources/hawtjni/native-package/src`. Created here files Makefile, config.h and modified `jansi_structs.c` for ispeed and ospeed termios getters and setters (files here [jansi_manual_build.zip](https://github.com/fusesource/jansi/files/3386256/jansi_manual_build.zip) ). Then run the make and voila: libjansi.so for Solaris is born. Then introduced it into jvm using `-Djava.library.path` option and works perfectly.
